### ISSUE TITLE
GENIE Training Diverged in LSUN Church

### ISSUE THREAD

**Issue by ZichenMiao** [nv-tlabs/GENIE#1]
Hi,

It's a great work and thank you for sharing the code!

Yet I met some problems in reproducing training GENIE on LSUN Church. I followed the instructions, used 8 x A5000, changed the training batch_size from 32 to 8, and lowered the learning rate from 1e-4 to 1e-5. And the model diverged. Below are samples from 1000, 2000, and 5000 iterations. 

![iter_1000](https://user-images.githubusercontent.com/63263970/228994991-e3a046ff-23e1-418a-b778-07df1f6c741a.png)
![iter_2000](https://user-images.githubusercontent.com/63263970/228994990-42f37675-4ee5-4944-8ca4-17c4e13011c8.png)
![iter_5000](https://user-images.githubusercontent.com/63263970/228994988-e2f0488b-e43f-4c70-9e41-b8b4e6a16d96.png)

I tried both lr=1e-4 and 1e-5. The results are the same.

**Comment by timudk** [comment]
Hi Zichen,

Thank you for opening the issue. The latest commit should fix the issue. I tested GENIE training on CIFAR-10 and it works again as expected. Let me know if you have any other issues.
